README.md

```diff
@@ -48,6 +48,7 @@ When learning CS there are some useful sites you must know to get always informe
    * [Dieter Rams : 10 Principles of Good Product Design](https://stephenhaunts.com/2013/12/11/dieter-rams-10-principles-of-good-product-design/)
    * [10-ways-to-be-a-better-developer](https://stephenhaunts.files.wordpress.com/2014/04/10-ways-to-be-a-better-developer.png)
    * [Working as a Software Developer](https://henrikwarne.com/2012/12/12/working-as-a-software-developer/)
+   * [Software design patterns and best practices](http://wiki.c2.com/)
```

Comment:
I'd call it the Portland Pattern Repository and use the description after the link.